AUTHORS

```diff
@@ -3310,5 +3310,17 @@ Mestery <mestery@pm.me>
 Himadri Ganguly <himadri.tech@gmail.com>
 Howie Zhao <howiezhaohr@hotmail.com>
 Luan Devecchi <luan@engineer.com>
+ashish maurya <ashishmaurya@outlook.com>
+Justin <justinleeong@gmail.com>
+Raz Luvaton <rluvaton@gmail.com>
+Don Jayamanne <don.jayamanne@yahoo.com>
+Felix Yan <felixonmars@archlinux.org>
+Ray <ray@isrc.iscas.ac.cn>
+Node.js GitHub Bot <github-bot@iojs.org>
```

Comment:
```suggestion
```

Reply:
> We decided it was OK to include the npm bot in AUTHORS, so I'm OK including other bots too. No bots except our bots, of course, but I'd include that in "our bots" if we ever had a situation where it managed to land a commit. (So far: nope.)

https://github.com/nodejs/node/pull/39858#issuecomment-904659297

Reply:
Optimistically marking as resolved. Please undo my resolution mark if that's wrong.